cs-office-365/src/main/java/io/cloudslang/content/office365/entities/GetMessageInputs.java

```diff
@@ -24,15 +24,17 @@
     private final String folderId;
     private final String selectQuery;
     private final String oDataQuery;
+    private final String filePath;
 
     private final Office365CommonInputs commonInputs;
 
-    @java.beans.ConstructorProperties({"messageId", "folderId", "selectQuery", "oDataQuery", "commonInputs"})
-    public GetMessageInputs(String messageId, String folderId, String selectQuery, String oDataQuery, Office365CommonInputs commonInputs) {
+    @java.beans.ConstructorProperties({"messageId", "folderId", "selectQuery", "oDataQuery", "filePath", "commonInputs"})
```

Comment:
I don't really understand why "filePath" was added to this class, as you're not using it for GetMessage - could you please explain?

Reply:
Thanks for the feedback, I have removed the input from the GetMessageInputs as it is not used within this class